F21/lorepap/utils.py

```diff
@@ -0,0 +1,54 @@
+import unicodedata
+
+
+class Word:
+
+  exceptionLanguages = ['tr', 'az', 'ga', 'el']
+  unchangedLanguages = ['ja', 'zh', 'th']
+
+  def __init__(self, word, bcp):
+    self.word = word
+    self.language = bcp
+    # self._standardIrishSpelling = std
+
+  def toLowercase(self):
+    """
+    Lowercase input word (self.word) according to language variable (self.language).
+    @param: Word object to access to the word variable (self.word)
+    @return: word string in its lowercase version
+    """
+
+    language = self.getLanguageFromBcp()
+    self.checkInvalidBcp(language)
+    tmp = self.word
+    if language in self.unchangedLanguages:
+        return tmp
+    elif language in self.exceptionLanguages:
+        if language == 'ga':
+            if len(self.word) > 1:
+                if (self.word[0] == 't' or self.word[0] == 'n') and unicodedata.normalize('NFC', self.word)[1] in 'AEIOU\u00c1\u00c9\u00cd\u00d3\u00da':
+                    tmp = self.word[0]+'-'+tmp[1:]
+        elif language == 'tr':
+            tmp = self.word
+            tmp = tmp.replace('\u0049', '\u0131')
+        elif language=='az':
+            tmp = self.word
+            tmp = tmp.replace('\u0049','\u0131')
+        elif language=='el':
+            if tmp[-1]=='\u03a3':
+                #self._finalSigma = True
+                tmp = tmp[:-1]+'\u03c2'
+    return tmp.lower()
```

Comment:
suggestion: instead of storing in "tmp" variable, you can just return it. This will omit the "tmp" variable